src/Core/Grid/Column/ColumnCollection.php

```diff
@@ -107,14 +107,39 @@ public function toArray()
         return $columns;
     }
 
+    /**
+     * Swap two existing columns.
+     *
+     * @param string $keyFrom the Column ID original position in the Collection
+     * @param string $keyTo the Column ID destination position in the Collection
+     *
+     * @return self
+     */
+    public function swap($keyFrom, $keyTo)
+    {
+        if (!array_key_exists($keyFrom, $this->items) || !array_key_exists($keyTo, $this->items)) {
```

Comment:
Wouldn't it be more performant to check `!isset($this->items[$keyFrom], $this->items[$keyTo])`?

Reply:
I do it first, and my tests were broken. I don't know why but I believe in tests more than in `isset()` function ;)